test/libsolidity/semanticTests/fallback/call_forward_bytes.sol

```diff
@@ -0,0 +1,26 @@
+contract receiver {
+    uint256 public received;
+    function recv(uint256 x) public { received += x + 1; }
+    fallback() external { received = 0x80; }
+}
+contract sender {
+    constructor() { rec = new receiver(); }
+    fallback() external { savedData = msg.data; }
+    function forward() public returns (bool) { address(rec).call(savedData); return true; }
+    function clear() public returns (bool) { delete savedData; return true; }
+    function val() public returns (uint) { return rec.received(); }
+    receiver rec;
+    bytes savedData;
+}
+// ====
+// compileToEwasm : false
+// compileViaYul: also
+// ----
+// (): 7 ->
+// val() -> 0
+// forward() -> true
+// val() -> 0x80
```

Comment:
Used to be 8

Reply:
From what I've understood from the logic of the test code, it should be 0 only.

The call to recv() will make way for a call to the fallback. That will merely store the savedData.
After that, the call to val() should be returning a 0 only, since that contract has not been touched by the control flow of the code yet